src/lib/TopLevel.hs

```
@@ -100,34 +81,19 @@ evalSourceBlockM env block = case sbContents block of
       val <- evalUModuleVal env v m
       case fmt of
         Printed -> do
-          logTop $ TextOut $ pprintVal val
-        Heatmap color -> logTop $ valToHeatmap color val
-        Scatter -> logTop $ valToScatter val
+          s <- liftIO $ pprintVal val
+          logTop $ TextOut s
+        Heatmap color -> return mempty -- logTop $ valToHeatmap color val
+        Scatter       -> return mempty -- logTop $ valToScatter val
```

Can we at least raise an error that explains what's wrong?